File: AlkmaarseRestaurants/angular/src/app/shared/models/Restaurant/RestaurantInfoModel.ts
import { IRestaurantInfoModel } from "./IRestaurantInfoModel";
import { RestaurantTypeModel } from "./RestaurantTypeModel";

export class RestaurantInfoModel implements IRestaurantInfoModel {
    public description: string;
    public phoneNumber: string;
    public mobilePhoneNumber: string;
    public website: string;
    public restaurantTypes: Array<RestaurantTypeModel>;

    constructor() {
        this.description = "";
        this.phoneNumber = "";
        this.mobilePhoneNumber = "";
        this.website = "";
        this.restaurantTypes = [];
    }
}

File: AlkmaarseRestaurants/Models/Restaurant/RestaurantInfoModel.cs
using System.Collections.Generic;

namespace AlkmaarseRestaurants.Models.Restaurant
{
    public class RestaurantInfoModel
    {
        public string Description { get; set; }
        public string PhoneNumber { get; set; }
        public string MobilePhoneNumber { get; set; }
        public string Website { get; set; }
        public List<string> RestaurantTypes { get; set; }
    }
}
